deploy/helm/sumologic/upgrade-1.0.0.sh

```diff
@@ -312,6 +315,64 @@ for i in $(seq 0 ${metrics_length}); do
     fi
 done
 
+# Fix fluent-bit env
+yq w -i new.yaml "fluent-bit.env(name==CHART).valueFrom.configMapKeyRef.key" "fluentdLogs"
+
+# Fix prometheus service monitors
+yq d -i "new.yaml" "prometheus-operator.prometheus.additionalServiceMonitors(name==collection-sumologic)"
+yq d -i "new.yaml" "prometheus-operator.prometheus.additionalServiceMonitors(name==collection-sumologic-otelcol)"
+yq d -i "new.yaml" "prometheus-operator.prometheus.additionalServiceMonitors(name==collection-sumologic-events)"
+echo '---
+prometheus-operator:
+  prometheus:
+    additionalServiceMonitors:
+      - name: collection-sumologic-otelcol
+        additionalLabels:
+          app: collection-sumologic-otelcol
+        endpoints:
+          - port: metrics
+        namespaceSelector:
+          matchNames:
+            - sumologic
+        selector:
+          matchLabels:
+            app: collection-sumologic-otelcol
+      - name: collection-sumologic-fluentd-logs
+        additionalLabels:
+          app: collection-sumologic-fluentd-logs
```

Comment:
These are dynamically generated no? based on the release and chart name the user uses?

Reply:
The labels and the namespace may change depending on where user installed?

Reply:
Yeah, you're right, we need to fix that
The simplest way is to ask the user to provide a namespace and helm release names as a script parameters I believe.

Reply:
Thats true 🤦

Reply:
however @frankreno these names are currently hardcoded in the `values.yaml` file for the prometheus service monitors, we might want to change it there if we're going to change it here

Reply:
I've updated the PR with commit for non-default release and namespace values; @samjsong wouldn't this work with hardcoded values as well?
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/605/files#diff-9d829ead41a2d6b7ad9d8bd01725aedbR328